popup/popupsPty.css

```diff
@@ -0,0 +1,207 @@
+.modalPty {
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 16px;
+  opacity: 1;
+}
+
+.cardPty {
+  background-color: #fff;
+  width: 343px;
+  height: 700px;
+  border-radius: 10px;
+  padding: 12px;
+  opacity: 1;
+  margin: auto;
+}
+
+.titlePty {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+}
+
+.closeButtPty {
+  color: #67798e;
+  font-size: 24px;
+  font-weight: 900;
+  position: absolute;
+  margin-left: 300px;
+  margin-top: -40px;
+  cursor: pointer;
+}
+
+.snapImages {
+  width: 100%;
+  height: 330px;
+}
```

Comment:
- [ ] You are doing well implementing the required changes 👍🏽  but I can see that the pop-up window doesn't look like the Figma design 🧐, it is because you are trying to show all the windows in the screen size, don't you? but the modal window is bigger than the screen, so please I think we could use `overflow-y: scroll` to allow the modal window to move vertically, kindly do the same in the mobile version because I cannot move to see all the modal for a better user interface.
- [ ] Additionally add the grey opacity margin around the modal window, in the desktop version there is no margin at the bottom and in the mobile version there are no margins on both sides to look similar to the Figma design.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/217711912-b912f0fd-3404-46a7-a839-155b06eb7117.png) | ![image](https://user-images.githubusercontent.com/58642949/217711822-eac34d16-c749-4f54-b66c-e266ff5f31e3.png) |
| ![image](https://user-images.githubusercontent.com/58642949/217712010-be01f6c3-8456-481f-a279-d44c832bc464.png) | ![image](https://user-images.githubusercontent.com/58642949/217711675-33007d1f-a7a5-4078-b54b-8eeae82bda15.png) |

Reply:
Okay